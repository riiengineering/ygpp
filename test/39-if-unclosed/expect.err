test/39-if-unclosed/input:3: error: unclosed #if*
