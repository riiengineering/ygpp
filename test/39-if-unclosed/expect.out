XYZ is defined as 42.
